### PR TITLE
PDF support

### PR DESCRIPTION
**Pull request by comptonsm**

### CODE REVIEW

**Review by gilbertEH — COMMENTED** [review]
I wonder if we need this in the main app too.

File: markymark/Classes/Renderer/ImageProcessors/MarkDownImageProcessor.swift [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+    // Convert input data/image to target image and return it.
+    func process(item: ImageProcessItem, options: KingfisherParsedOptionsInfo) -> KFCrossPlatformImage? {
+        switch item {
+        case .image(let image):
+            return image.kf.scaled(to: options.scaleFactor)
+        case .data(let data):
+            // If we have a known/supported image format then defer to standard data processing
+            guard data.kf.imageFormat == .unknown else {
+                return KingfisherWrapper.image(
```
[the remark this answers is not in the document]
Comment (comptonsm) [reply markymark/Classes/Renderer/ImageProcessors/MarkDownImageProcessor.swift:26]: I tried that but `'imageCreatingOptions' is inaccessible due to 'internal' protection level`

**Comment by comptonsm** [comment]
> I wonder if we need this in the main app too.

That is a good point. I think this is pretty hight priority for our 1.0.1 release so I'd rather make that change later. I guess we could add configuration options for Kingfisher to this library (including ImageProcessor) and then we'd pass them through from the main app. Is that what you are thinking?

**Comment by gilbertEH** [comment]
> > I wonder if we need this in the main app too.
> 
> That is a good point. I think this is pretty hight priority for our 1.0.1 release so I'd rather make that change later. I guess we could add configuration options for Kingfisher to this library (including ImageProcessor) and then we'd pass them through from the main app. Is that what you are thinking?

Yep, something similar to that.

**Review by gilbertEH — APPROVED** [review]

**Comment by comptonsm** [comment]
Closing and reopening as #9. The commit history got messed up from the squash & merge and auto base branch change here.

**Pull request closed without merging**